data-prepper-core/src/main/java/com/amazon/dataprepper/DataPrepper.java

```diff
@@ -55,6 +57,15 @@ public static void configureWithDefaults() {
         configureMeterRegistry();
     }
 
+    /**
+     * returns serviceName if exists or default serviceName
+     * @return serviceName for data-prepper
+     */
+    public static String getServiceNameForMetrics() {
+        final String serviceName = System.getenv(DATAPREPPER_SERVICE_NAME);
+        return (serviceName != null && !serviceName.trim().isEmpty()) ? serviceName : DEFAULT_SERVICE_NAME;
```

Comment:
nit - micrometer (and a bunch of other dependencies) provide a `StringUtils.isBlank()` method for the same purpose